### DISCUSSION TITLE
"Community Standards" are repo metrics?

### DISCUSSION THREAD

**Discussion by frenck** [community/OSPO#71] [Dashboard]
When click for more metric information here:

![CleanShot 2023-03-21 at 16 23 55](https://user-images.githubusercontent.com/195327/226654641-26a3ab47-37ac-4c6d-a052-fb72c4c0fcd6.png)

I get presented with graphs, and a large block on top with this:

![CleanShot 2023-03-21 at 16 22 00](https://user-images.githubusercontent.com/195327/226654301-5142f4e4-eff1-44b0-bc4c-1fef4582e128.png)

I'm confused. Why is that there? How does that information relate to the metrics shown?

**Comment by ahpook** [comment]
It's a fair point. They are at the top of the flyout because that's where the "detail view" for an individual repo lives; there's no other place currently to show that kind of detail. An earlier version of the data table showed how many of docs were present in each repo, which was a useful metric: "Show me all the repos with a missing README". The field isn't displayed but you can still filter on them, with the `has:` and `license:` syntax:

<img width="564" alt="Screen Shot 2023-03-21 at 3 43 38 PM" src="https://user-images.githubusercontent.com/56753/226758216-abe8ad83-ce09-4766-9d9a-59b02e1dbf15.png">
